tutorials/index.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Tutorials
+suppressRelated: true
+---
+
+These tutorials will give you step-by-step instruction to help you learn NServiceBus.
+
+### [NServiceBus 101: Messaging Basics](nservicebus-101/)
+
+In this course, you will learn how to get working quickly with NServiceBus, including how to set up your development environment and create messaging endpoints, defining messages and handlers, how to send messages and publish events, and how to deal with exceptions and take advantage of automatic retries.
+
+### Additional courses
+
+Please [sign up for updates](https://particular.net/s/new-course-signup) if you'd like to be notified when we release additional courses.
```

Comment:
@weralabaj i am not saying we shouldnt do it. i am saying if it makes sense we should do it in a way that the value is delivered for all pages and in a way that is consistent with how docs is managed as a whole. but as it stands the current approach is nether of those things.